docs/administration/lokistack.adoc

```diff
@@ -0,0 +1,399 @@
+
+== Configure LokiStack using S3 Bucket
+
+=== LokiStack Operator
+* Install the Red Hat Loki Operator from the OperatorHub
+
+=== Storage Bucket
+* Create an https://docs.openshift.com/container-platform/4.16/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage-aws_installing-log-storage[AWS S3 bucket] or https://docs.openshift.com/container-platform/4.16/observability/logging/log_storage/installing-log-storage.html#logging-loki-storage-aws_installing-log-storage[other] storage object
++
+.Command to create a bucket named *my-bucket-logging-loki* in region *us-east-1*
+[source]
+----
+  aws s3api create-bucket --acl private --region us-east-1 --bucket my-bucket-logging-loki
+----
++
+.Addtional option required for any region other than *us-east-1*
+----
+  --create-bucket-configuration LocationConstraint="$REGION"
+----
+NOTE: Bucket names must be unique and should be prepended with an easily identifiable term (i.e. username or cluster_id)
++
+Access Control::
+AWS recommends that you turn on all options to https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html?icmpid=docs_amazons3_console[block public access] for buckets
++
+.Block configurations recommended
+----
+  BlockPublicAcls=true
+  IgnorePublicAcls=true
+  BlockPublicPolicy=true
+  RestrictPublicBuckets=true
+----
+
+=== Storage Secret
+* Create a secret for the lokistack instance (https://docs.openshift.com/container-platform/4.16/observability/logging/log_storage/installing-log-storage.html#loki-create-object-storage-secret-console_installing-log-storage[docs])
++
+.Secret named *logging-loki-s3*
+[source]
+----
+  oc create -n openshift-logging secret generic logging-loki-s3 \
+    --from-literal=region="us-east-1" \
+    --from-literal=bucketnames="my-bucket-logging-loki" \
+    --from-literal=access_key_id="AKIAIOSFODNN7EXAMPLE" \
+    --from-literal=access_key_secret="wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY" \
+    --from-literal=endpoint="https://s3.us-east-1.amazonaws.com"
+----
++
+or
++
+[source,yaml]
+----
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: logging-loki-s3
+      namespace: openshift-logging
+    stringData:
+      access_key_id: AKIAIOSFODNN7EXAMPLE
+      access_key_secret: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+      bucketnames: my-bucket-logging-loki
+      endpoint: https://s3.us-east-1.amazonaws.com
+      region: us-east-1
+----
+
+=== LokiStack Instance
+* Create an instance of lokistack
++
+.LokiStack
+[source,yaml]
+----
+    apiVersion: loki.grafana.com/v1
+    kind: LokiStack
+    metadata:
+      name: logging-loki
+      namespace: openshift-logging  # <1>
+    spec:
+      size: 1x.extra-small # <2>
+      storage:
+        schemas:
+        - version: v12
+          effectiveDate: 2022-06-01
```

Comment:
`v13` is the latest schema version ... and let's also use a more recent date, so that we see who copy&pasted from this example :wink: 

```suggestion
        - version: v13
          effectiveDate: 2024-10-23
```